plugin/case-reporting/pom.xml

```diff
@@ -8,4 +8,12 @@
 	</parent>
 
 	<artifactId>cqf-ruler-case-reporting</artifactId>
+	<dependencies>
+		<!-- https://mvnrepository.com/artifact/commons-beanutils/commons-beanutils -->
+		<dependency>
+			<groupId>commons-beanutils</groupId>
+			<artifactId>commons-beanutils</artifactId>
+			<version>1.9.4</version>
+		</dependency>
+	</dependencies>
```

Comment:
using this to follow the changelog paths, I tried to use FhirPath but that ended up being complicated